### ISSUE TITLE
%{value} in format message

### ISSUE THREAD

**Issue by nyrsimon** [ansman/validate.js#32]
Firstly - awesome stuff you are building here, it's a big help!!

Am I doing something wrong - I'm trying to include the value of the field being tested in the custom error message. This works fine on inclusion but not on the 'format' check.

For example, this works great
```
currency: {
	presence: true,
	length: {is: 3},
	inclusion: {
		within: ccys,
		message: '%{value} is not a valid currency'
	},
	format: {
		pattern: /^[A-Z]{3}$/,
		message: 'is in an invalid format'
	}
}
```
But if I go with this
```
transaction_time: {
	presence: true,
	length: {is: 8},
	format: {
		pattern: /^[0-2][0-3]:[0-5][0-9]:[0-5][0-9]$/,
		message: '%{value} is in an invalid format '
	}
}
```
I get 'Transaction time %{value} is in an invalid format'

Am I doing something wrong or is it just not a feature at this point??

Thanks

**Comment by ansman** [comment]
Not implemented yet, but it's a good idea. I'll make sure to add it to the backlog!

**Comment by nyrsimon** [comment]
OK thanks - I'll try to take a look and see if I can cook up the change. Would love to add something back

**Comment by nyrsimon** [comment]
So took a look. I ended up changing line 740 from
```
return message;
```

to
```
return v.format(message, {value: value});
```

**Comment by ansman** [comment]
Well, perhaps we should start supporting adding values to all messages in which case this should be done on a higher level. I'll think about the implementation.

**Comment by ansman** [comment]
Released in 0.7.0, all validators now support replacing `%{format}`
